### ISSUE TITLE
window_opacity don't take effect

### ISSUE THREAD

**Issue by blackpeter11** [koekeishiya/yabai#1283]
Question:

Whether I type in the command“yabai -m config window_opacity on" like this:
<img width="437" alt="image" src="https://user-images.githubusercontent.com/90673507/170768317-0284e3a2-aeac-4622-8b00-2a6ba757b3df.png">

or load scripts like this:
<img width="442" alt="image" src="https://user-images.githubusercontent.com/90673507/170768596-9d5a9bfe-ab4d-450f-aa36-bd2e6f0a1771.png">

None of the transparency effects will take effect.

My configuration：

```
# bsp or float (default: float)
yabai -m config layout bsp

# Override default layout for space 2 only
yabai -m config --space 2 layout float

# Set all padding and gaps to 20pt (default: 0)
/* yabai -m config top_padding    5 */
/* yabai -m config bottom_padding 5 */
/* yabai -m config left_padding   5 */
/* yabai -m config right_padding  5 */
yabai -m config window_gap     10

# Override gaps for space 2 only
yabai -m config --space 2 window_gap 0

# on or off (default: off)
yabai -m config auto_balance on

# Floating point value between 0 and 1 (default: 0.5)
yabai -m config split_ratio 0.5

# set mouse interaction modifier key (default: fn)
yabai -m config mouse_modifier fn

# set modifier + left-click drag to move window (default: move)
yabai -m config mouse_action1 move

# set modifier + right-click drag to resize window (default: resize)
yabai -m config mouse_action2 resize

# set mouse follows focus mode (default: off)
yabai -m config mouse_follows_focus off

# floating windows are always on top (default: off)
yabai -m config window_topmost on

# modify window shadows (default: on, options: on, off, float)
# example: show shadows only for floating windows
yabai -m config window_shadow float

# set focus follows mouse mode (default: off, options: off, autoraise, autofocus)
yabai -m config focus_follows_mouse autoraise

# window opacity (default: off)
# example: render all unfocused windows with 90% opacity 
yabai -m config window_opacity on
yabai -m config window_opacity_duration 0.0
yabai -m config active_window_opacity 1.0
yabai -m config normal_window_opacity 0.8

yabai -m config window_border                on
yabai -m config active_window_border_color   0xff2c9678
yabai -m config normal_window_border_color   0xfff9d367
yabai -m config insert_feedback_color        0xffee3f4d
```

<img width="668" alt="image" src="https://user-images.githubusercontent.com/90673507/170769729-29629333-1ba6-455e-ace7-385fe8f4538a.png">

My envrionment:
M1 pro (mac os monterey 12.3.1)
yabai version 4.0.1

**Comment by madkinsz** [comment]
Is the scripting additional installed?  See https://github.com/koekeishiya/yabai/wiki/Installing-yabai-(latest-release)#macos-big-sur-and-monterey---automatically-load-scripting-addition-on-startup

You may want to follow some of the debug steps in https://github.com/koekeishiya/yabai/issues/1054 to obtain more information if the scripting addition is not installing correctly.

**Comment by blackpeter11** [comment]
您好，这里是杜炳霖，您的邮件我已经收到了，我会尽早阅读后给您回复。如果事出紧急，请联系电话18944525557。

**Comment by whexy** [comment]
The same thing happened on my M1 Pro (macOS 12.4, yabai 4.0.4). I think the question is related to arm64e ABI.

Solved following the steps mentioned in #1054, with `sudo nvram boot-args=-arm64e_preview_abi`.

**Comment by blackpeter11** [comment]
您好，这里是杜炳霖，您的邮件我已经收到了，我会尽早阅读后给您回复。如果事出紧急，请联系电话18944525557。
